content/en/catalyst/fund9/challenges/legal-and-finacial-implementation/campagin-brief.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "legal & finacial implementation"
```

Comment:
Should be written, 'Legal & Financial Implementation'

Reply:
Hi Jack @PardusEidolon 
Final fixing for this typo :) then we can ship to the main branch